### PR TITLE
translation / context in browse

### PR DESCRIPTION
**Pull request by BT-ojossen**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you please clean a bit the commit history?

**Comment by luc-demeyer** [comment]
I ,think this PR should be closed since
- translation of po file is done via PR #119, so it's recommended to update this PR to have both pdf and xls translations.
- the code to pass the context is not correct, cf.PR #123

**Comment by pedrobaeza** [comment]
https://github.com/OCA/account-financial-reporting/pull/123 is a patch for solving the same problem, please judge which one is best.

File: account_financial_report_webkit/i18n/de.po [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # German translation for account-financial-report
 # Copyright (c) 2014 Rosetta Contributors and Canonical Ltd 2014
 # This file is distributed under the same license as the account-financial-report package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2014.
+# FIRST AUTHOR <olivier.jossen@braintec-group.com>, 2014.
```
Comment (pedrobaeza) [inline account_financial_report_webkit/i18n/de.po:4]: Why changing the translation in the same PR? Both changes must be separate PR

**Comment by BT-ojossen** [comment]
Please merge #123
